mavlink/mavManager.js

```diff
@@ -213,17 +215,22 @@ class mavManager {
     this.udpStream.bind(this.inudpPort, this.inudpIP)
   }
 
-  sendData (msg) {
+  sendData (msg, vehicle = 255, component = 1) {
```

Comment:
Since we can safely assume the Rpanion will always be on the vehicle, just use ``this.targetSystem`` instead of an argument in the function.

Reply:
Done